media-api/app/lib/ImageResponse.scala

```diff
@@ -132,8 +153,11 @@ object ImageResponse {
     __.json.update(__.read[JsObject].map(_ ++ Json.obj("cost" -> cost.toString)))
   }
 
-  def addPersistedState(isPersisted: Boolean): Reads[JsObject] =
-    __.json.update(__.read[JsObject]).map(_ ++ Json.obj("persisted" -> isPersisted))
+  def addPersistedState(isPersisted: Boolean, persistenceReasons: List[String]): Reads[JsObject] =
+    __.json.update(__.read[JsObject]).map(_ ++ Json.obj(
+      "persisted" -> Json.obj(
+        "data" -> isPersisted,
```

Comment:
Is `"data"` the best name? Seems to allude to it being a sub-resource, but it's not. Would `"value"` or `"state"` be clearer?